src/App.vue

```diff
@@ -15,7 +15,7 @@
             return {
                 authenticated: false,
                 mockAccount: {
-                    username: "admin",
+                    email: "admin",
                     password: "admin"
```

Comment:
We'd like to avoid adding any user info into code, even if it is mock data. Can we remove this and add instructions to the README instead?

Reply:
this isn't even used anymore, deleting